### PR TITLE
Adding day1 MachineConfig utility

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
Adding standalone Day1 utility. Allows manually producing day1 MachineConfig.

### CODE REVIEW

File: cmd/day1-utility/main.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		kernelModule = flag.String("kernelModule", "", "kernel module that should be loaded by MachineConfig")
+	)
+
+	flag.Parse()
+
+	if flag.NFlag() != 4 {
+		fmt.Println("Not enough parameters are provided", "num parameters", flag.NFlag(), "git commit", GitCommit, "version", Version)
+		flag.Usage()
+		os.Exit(1)
+	}
```
Comment (qbarrand) [inline cmd/day1-utility/main.go:30]: If those values are mandatory, then they should be arguments, not flags.

Comment (yevgeny-shnaidman) [reply cmd/day1-utility/main.go:30]: flags package does not support mandatory parameters. By checking the number of flag we are actually making them mandatory

File: cmd/day1-utility/main.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	fmt.Fprintf(os.Stderr, "Arg0:\n\tname of the MachineConfig to be created\n")
+	fmt.Fprintf(os.Stderr, "Arg1:\n\tMachineConfigPool name that should be referenced by MachineConfig\n")
+	fmt.Fprintf(os.Stderr, "Arg2:\n\tkernel module name, that should be unloaded (in-tree) and then loaded (oot)\n")
+	fmt.Fprintf(os.Stderr, "Arg3:\n\tcontainer image that contains kernel module .ko file\n\n")
+	fmt.Fprintf(os.Stderr, "Examlpe: day1-utility <machine config name> <machine config pool name> <kernel module> <kernel module image>\n")
+}
+
+func main() {
+	flag.Parse()
+	flag.Usage = customUsage
```
[the remark this answers is not in the document]
Comment (yevgeny-shnaidman) [reply cmd/day1-utility/main.go:27]: Fixed

**Pull request merged**